### DISCUSSION TITLE
guide/optimize-build/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#395] [docs-comment]
# guide/optimize-build/

图片优化 当我们在 markdown 中使用 [alt](url) 或者 <img src="url"> 嵌入图片后，虽然页面按照预期的显示了 图片。 由于图片的体积不同，当图片体积较小，网络情况良好的时候，我们不会感受到页面的布局有产生明显的抖动。 然而，当图片体积较大，或者网络情况较差时，由于图片为完成加载，原本页面上应该显示图片的位置被后面的 内...

https://theme-plume.vuejs.press/guide/optimize-build/

**Comment by Mars13333** [comment]
你好，我在打包的时候，产生了一条警告。请问一下，这个警告有什么优化方式么：
(!) Some chunks are larger than 2048 kB after minification. Consider:
- Using dynamic import() to code-split the application
- Use build.rollupOptions.output.manualChunks to improve chunking: https://rollupjs.org/configuration-options/#output-manualchunks
- Adjust chunk size limit for this warning via build.chunkSizeWarningLimit.

**Reply by pengzhanbo** [reply]
忽略即可，此警告仅是提示单个文件提交超过 2M。但这并不会造成使用上的问题。如果想要优化，可以参考 vite 的文档进行配置。

**Comment by Mars13333** [comment]
不好意思，再打扰一下。我升级之后遇到了部署问题，很费解。
在github page设置中，设置branch为root才能上传并通过action编译，但是无法访问github.io。必须换成docs才能访问github.io,但是在docs情况下，无法上传并通过编译，会action中编译失败。日志大概如下：
 Logging at level: debug GitHub Pages: github-pages v232 GitHub Pages: jekyll v3.10.0 Theme: jekyll-theme-primer Theme source: /usr/local/bundle/gems/jekyll-theme-primer-0.6.0 Requiring: jekyll-github-metadata Requiring: jekyll-seo-tag Requiring: jekyll-coffeescript Requiring: jekyll-commonmark-ghpages Requiring: jekyll-gist Requiring: jekyll-github-metadata Requiring: jekyll-paginate Requiring: jekyll-relative-links Requiring: jekyll-optional-front-matter Requiring: jekyll-readme-index Requiring: jekyll-default-layout Requiring: jekyll-titles-from-headings GitHub Metadata: Initializing... Source: /github/workspace/./docs Destination: /github/workspace/./docs/_site Incremental build: disabled. Enable with --incremental Generating... Generating: JekyllOptionalFrontMatter::Generator finished in 1.2614e-05 seconds. Generating: JekyllReadmeIndex::Generator finished in 6.061e-06 seconds. Generating: Jekyll::Paginate::Pagination finished in 3.707e-06 seconds. Generating: JekyllRelativeLinks::Generator finished in 2.664e-05 seconds. Generating: JekyllDefaultLayout::Generator finished in 1.1442e-05 seconds. Generating: JekyllTitlesFromHeadings::Generator finished in 6.662e-06 seconds. Rendering: assets/css/style.scss Pre-Render Hooks: assets/css/style.scss Rendering Markup: assets/css/style.scss github-pages 232 | Error: No such file or directory @ dir_chdir0 - /github/workspace/docs

**Reply by Mars13333** [reply]
ok了。之前低版本好久没升级过，我在config.ts中加了base: '/' 就好了。设置root即可，能上传，也能编译通过和访问了。
